source/2020-06-05-the-ember-times-issue-151.md

```diff
@@ -104,14 +105,15 @@ Read more on the full the blog posts on [using Oauth2 with Flask-REST-JSONAPI](h
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Blog post: How to integrate Netlify-CMS with Empress-blog 🚀](https://www.mylittletechlife.com/integrate-netlify-cms-with-empress-blog)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Robin Philip Thomas](https://github.com/MalayaliRobz) wrote a blog on integrating empress-blog and Nelify CMS.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+[empress-blog](https://github.com/empress/empress-blog) is designed to be a fully-functional, static site implementation of a blog system built on Ember with fully working out of the box SEO friendly output based on markdown. [Netlify CMS](https://www.netlifycms.org/) is based on client-side javascript, reusable APIs and prebuilt markup.
```

Comment:
I'm not too familiar with Empress so I wasn't sure how to parse the first sentence. I wondered if making a list might help clarify what its features are.

If I understand you right, maybe:
```suggestion
What is [empress-blog](https://github.com/empress/empress-blog)?

- A fully-functional, static site implementation of a blog system
- Built on Ember
- Fully working out of the box
- SEO friendly output based on Markdown

[Netlify CMS](https://www.netlifycms.org/) is based on client-side JavaScript, reusable APIs and prebuilt markup.
```

Reply:
Ah! That sounds perfect @ijlee2 :)